security-policies/bundle/compliance/cis_azure/rules/cis_9_10/data.yaml

```diff
@@ -0,0 +1,104 @@
+metadata:
+  id: 43d5538c-17a3-5e04-9c06-ad4323bfd188
+  name: Ensure FTP deployments are Disabled
+  profile_applicability: '* Level 1'
+  description: |-
+    By default, Azure Functions, Web, and API Services 
+    can be deployed over FTP.
+    If FTP is required for an
+    essential deployment workflow, FTPS should be required
+    for FTP login for all App Service Apps and Functions.
```

Comment:
Why are there randomly placed new lines? I wonder I it looks in Kibana

Reply:
This is auto-generated from the CIS provided benchmarks, let's see and it if it problematic we will need to change the behavior of parsing the CIS benchmark